docs/object.md

```diff
@@ -136,6 +146,7 @@ Function Name | Description | Return | &nbsp;
 getAttachments() | Returns a table in the same format as [getObjects()](#getobjects) for containers. | [<span class="ret tab"></span>](types.md) |
 <a class="anchor" id="getcolortint"></a>getColorTint() | Color tint. | [<span class="ret col"></span>](types.md#color) |
 getCustomObject() | Returns a Table with the Custom Object information of a Custom Object. | [<span class="ret tab"></span>](types.md) | [<span class="i"></span>](#getcustomobject)
+<a class="anchor" id="getdescription"></a>getDescription() | Same thing as getJson() except works with Tables. Works with [spawnObjectData()](base.md#spawnobjectdata). | [<span class="ret tab"></span>](types.md)
```

Comment:
```
-<a class="anchor" id="getdescription"></a>getDescription() | Same thing as getJson() except works with Tables. Works with [spawnObjectData()](base.md#spawnobjectdata). | [<span class="ret tab"></span>](types.md)
+<a class="anchor" id="getdata"></a>getData() | Returns a plain data representation of the object. Same as [getJson()](#getjson) except that a data structure made of nested tables is returned, rather than a string. Works with [spawnObjectData()](base.md#spawnobjectdata). | [<span class="ret tab"></span>](types.md)
```

I think the above diff would be ideal.

Reply:
If it is alphabetical like the rest of the them wouldn't getData come first?

Reply:
Sorry, the above represents a "diff", subtraction/addition as the first character on the line. So what I meant by that was to delete the old line, and replace it with the new one. I'll fix this up locally - I'd actually already made this change 😉